### PR TITLE
Project popup menu

### PR DESCRIPTION
**Pull request by henry-dura**
We created a popup menu that displays when the website user clicks "see project".  This will show you the details of the project.

### CODE REVIEW

**Comment by mirouhml** [comment]
Hi @henry-dura @Tobby8629, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have a complete README, please know that you have to add your partner's information in the Author's section and you have to modify the section Future Features, by including 1-3 potential features to be added.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: style.css [hunk 1]
```diff
@@ -415,6 +420,65 @@ hr.mobile {
...
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  z-index: 999;
+  padding: 5px 12px 24px 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
+  width: 100%;
+
```
Comment (thecodechaser) [inline style.css]: - [ ] You did a great job on the styles but for this project, you need to stick to the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471&t=Je0IY5iv4NIVovwd-0) template as much as possible. Please fix the below issues and make sure your design matches the Figma design.
  - Please increase the height of the model and increase the gap between project content.
  - Please make sure the model background is blurred.
  - Please increase font weight for project technologies.
  - Please make sure the model buttons are aligned to the bottom of the model as on the Figma design.

### Your design

image

### Figma design

image

File: style.css [hunk 2]
```diff
@@ -415,6 +420,65 @@ hr.mobile {
...
+.modal-mobile,
+.modal-desktop {
+  position: fixed;
+  top: 0;
+  background-color: #fff;
+  z-index: 999;
+  padding: 5px 12px 24px 12px;
+  display: flex;
+  flex-direction: column;
+  gap: 8px;
```
Comment (thecodechaser) [inline style.css]: - [ ] Please make sure to add top and left-right margins to the model for the mobile screens.

### Your design

image

### Figma design

image

File: style.css [hunk 3]
```diff
@@ -415,6 +420,65 @@ hr.mobile {
...
+  width: 100%;
+
+  /* box-shadow: rgba(0, 0, 0, 0.35) 0px 5px 15px; */
```
Comment (thecodechaser) [inline style.css]: - [ ] Please remove commented code from the file, It's not good practice to leave commented code in the files.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @henry-dura, and @Tobby8629👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -691,4 +770,53 @@ hr.mobile {
...
+
+  .modal-desktop {
+    width: 70%;
+    top: 10%;
+    left: 15%;
+    padding: 27px 24px;
+    gap: 50px;
+  }
+
+  .modal-headings {
```
Comment (GedeonTS) [inline style.css]: - [ ] Good job implementing the changes required by the first reviewer, :clap: unfortunately you have introduced some small issues that need to be addressed. Please make sure you add an overlay background that is dark and blur as per the following design.

-Your design:
YourOverlay
-Figma design:
figmaOvarlay

- [ ] Please make sure the text does not exceed the width of the popup
yourDesktopExceed

Comment (GedeonTS) [reply style.css]: - [ ] I think It is necessary for the user to see the button below your popup please make sure the screen is scrollable. Please add an overflow scroll to the overlay

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @HanaSabih , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights :tophat: :+1: 
- Good use of GitHub flow :+1: 
- Linters are green :+1: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 
- [OPTIONAL]  Kindly make sure you write a good descriptive PR summary and title. Describing the steps you followed to complete this milestone.
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**